mlrun/common/model_monitoring/helpers.py

```diff
@@ -72,7 +73,10 @@ def parse_monitoring_stream_path(
                 "Custom kafka topic is not allowed"
             )
         # Add topic to stream kafka uri
-        if application_name is None:
+        if (
+            application_name is None
+            or application_name == mm_constants.MonitoringFunctionNames.STREAM
+        ):
```

Comment:
In that case, I think renaming `application_name` to `function_name`, as the monitoring stream is not an application in out terminology